depmapomics/qc/test_celllines.py

```diff
@@ -51,7 +51,7 @@ def test_unexpected_arxspans(arxspans, omicssource, portal):
     if IGNORE_FAILED_TO_RELEASE:
         failed_to_release = set()
     else:
```

Comment:
shouldn't we merge the pytests and the depmapomics/qc?

Reply:
II'm not sure what the pytests are, but I'm expecting there are two types of tests that make sense:

1. QC tests which get run on the real data and flag for review anything that looks suspicious. Every release may have a small number of QC failures which get investigated and someone makes a decision whether they're okay or not.

2. Automated tests which don't use real data and are run to verify that the code is functioning as designed and no regressions are introduced after changes to the code are made.

Are the "pytests" automated tests or are they also "QC" like things?